scripts/task/KvToRdsMigration.php

```diff
@@ -0,0 +1,274 @@
+                    );
+                } else {
+                    $this->rdsStorage->storeTestVariable(
+                        $deliveryExecutionIdentifier,
+                        $variable->test,
+                        $variable->variable,
+                        $variable->callIdTest
+                    );
+                }
+            }
+            $count++;
+        }
+
+        if ($count == 0) {
+            $message = 'Already migrated.';
+        } else {
+            $message =  $count . ' variables migrated';
+        }
+
+        return \common_report_Report::createInfo('Migrating ' . $callId . ' : ' . $identifier . ' : ' . $message);
+    }
+
+    /**
+     * Get the configured service to deal with result storage
+     *
+     * @return \taoResultServer_models_classes_WritableResultStorage
+     * @throws \common_exception If the service is not a key value interface
+     */
+    protected function getCurrentKvResultStorage()
+    {
+        /** @var ResultServerService $resultService */
+        $resultService = $this->getServiceLocator()->get(ResultServerService::SERVICE_ID);
+        $resultStorageKey = $resultService->getOption(ResultServerService::OPTION_RESULT_STORAGE);
+        if ($resultStorageKey != 'taoAltResultStorage/KeyValueResultStorage') {
+            throw new \common_Exception('Result storage is not on KeyValue storage mode.');
+        }
+        return $resultService->instantiateResultStorage($resultStorageKey);
+    }
+
+    /**
+     * Register rdsStorage as default result storage
+     *
+     * @throws \common_Exception
+     */
+    protected function setCurrentResultStorageToRdsStorage()
+    {
+        if ($this->isDryrun()) {
+            return;
+        }
+        /** @var ResultServerService $resultService */
+        $resultService = $this->getServiceLocator()->get(ResultServerService::SERVICE_ID);
+        $resultService->setOption(ResultServerService::OPTION_RESULT_STORAGE, RdsResultStorage::SERVICE_ID);
+        $this->registerService(ResultServerService::SERVICE_ID, $resultService);
+    }
+
+    /**
+     * Get the delivery execution service
+     *
+     * @return ServiceProxy
+     */
+    protected function getDeliveryExecutionService()
+    {
+        return $this->getServiceLocator()->get(ServiceProxy::SERVICE_ID);
+    }
+
+    /**
+     * Display help
+     *
+     * @return array
+     */
+    protected function provideUsage()
+    {
+        return [
+            'prefix' => 'h',
+            'longPrefix' => 'help',
+            'description' => 'Prints a help statement'
+        ];
+    }
+
+    /**
+     * Provides script description
+     *
+     * @return string
+     */
+    protected function provideDescription()
+    {
+        return __('A script to migrate result from KeyValue storage to RDS.' . PHP_EOL .
+            ' It copies data from KeyValue to RdsStorage and switch config from taoResultServer/resultservice.conf.php.');
```

Comment:
While it's good that you mention the config will be updated, it might be good to be more specific here, by saying something like:

`It copies data from KeyValue to RdsStorage and updates the config in taoResultServer/resultservice.conf.php to use RdsResultStorage`